metrics/configs/flakes-daily-config.yaml

```diff
@@ -0,0 +1,109 @@
+metric: flakes-daily
```

Comment:
I'm not sure where that config value would be used. I don't think that the job has a way to display the description usefully so it would mainly be an annotation of the yaml config itself. There is already a brief description of each metric in the README (enforced by unit test). Should I expand on those descriptions?

Reply:
It would be used by the human being who reads the file